command.go

```diff
@@ -1176,6 +1176,30 @@ func (c *Command) ValidateArgs(args []string) error {
 	return c.Args(c, args)
 }
 
+// RequiredFlagError represents a failure to validate required flags.
+type RequiredFlagError struct {
+	Err error
+}
+
+// Error satisfies the error interface.
+func (r *RequiredFlagError) Error() string {
+	return r.Err.Error()
+}
+
+// Is satisfies the Is error interface.
+func (r *RequiredFlagError) Is(target error) bool {
+	err, ok := target.(*RequiredFlagError)
+	if !ok {
+		return false
+	}
+	return r.Err == err
+}
+
+// Unwrap satisfies Unwrap error interface.
+func (r *RequiredFlagError) Unwrap() error {
+	return r.Err
+}
```

Comment:
This seems to be overkill.

I'm doubtful about the implementation.

 I'm not sure you need a structure, especially while the type is a simple wrapper here.

If you had stored the fields name, I would have understood, but here you don't.

Also I will have double check about the need of `Is` method. Usually, adding Unwrap is enough.

I will come back to you later with feedbacks and maybe another implementation